docs/standard/choosing-core-framework-server.md

```diff
@@ -91,7 +91,7 @@ Some .NET Framework technologies are not available in .NET Core. Some of them wi
 
 * Windows Presentation Foundation (WPF) and Windows Forms: WPF and Windows Forms applications are only available on the .NET Framework. There are no plans to port them to .NET Core. 
 
-* Language support: Visual Basic and F# don’t currently have tooling support .NET Core, but both will be supported in Visual Studio 2017 and later versions of Visual Studio.
+* Language support: Visual Basic and F# don’t currently have tooling support for .NET Core, but both will be supported in Visual Studio 2017 and later versions of Visual Studio.
```

Comment:
Thanks @scottaddie but this statement is actually outdated. We do have tooling support for the 3 languages (not for ASP.NET Core though). Do you mind rewriting this?